src/pages/Earn/Earn.tsx

```diff
@@ -213,7 +214,11 @@ const Earn: React.FC<EarnProps> = ({ version, stakingInfos, poolMap }) => {
                   ...stakingInfo
                 }))
             } else {
-              return fetch(`https://api.pangolin.exchange/pangolin/apr/${stakingInfo.stakingRewardAddress}`)
+              return fetch(
+                `https://api.pangolin.exchange/pangolin/apr/${
+                  stakingInfo.stakingRewardAddress[chainId || ChainId.AVALANCHE]
```

Comment:
can you please make sure that `stakingInfo.stakingRewardAddress[chainId || ChainId.AVALANCHE]` is correct because in `useStakingInfo` , `stakingRewardAddress` is simply string

Reply:
After our modifications stakingInfo.stackingRewardAddress returns an object that corresponds to the MINICHEF_ADDRESS object for farming v2, which we have made dynamic. In https://github.com/pangolindex/interface/blob/dev/src/state/stake/hooks.ts#L1055 we can see the assignment of MINICHEF to stackingRewardAddress. However, for singleSideStaking (https://github.com/pangolindex/interface/blob/dev/src/pages/Beta/Stake/ClaimWidget/index.tsx#L33) we have to remove [chainId || ChainId.AVALANCHE] because for the moment there is no staking on WAGMI. I'll make a PR to correct it.

Reply:
Also just to be sure, FARMS_V0/V1 are not up anymore?

Reply:
yes we dont have farm v0 and v1 now. but users who are not migrated to farm v2, will still see those old farms.